util-test/src/main/scala/com/twitter/util/testing/ArgumentCapture.scala

```diff
@@ -0,0 +1,115 @@
+package com.twitter.util.testing
+
+import java.util.{List => JList}
+import org.mockito.ArgumentCaptor
+import org.mockito.exceptions.Reporter
+import scala.collection.JavaConversions._
```

Comment:
in general, we prefer JavaConverters to JavaConversions.  If you want to avoid the extra allocation, could you invoke it explicitly?